R/create.R

```diff
@@ -394,9 +394,10 @@ create_gp_data <- function(gp = gp_opts(), data) {
     ls_max = data$t - data$seeding_time - data$horizon,
     alpha_sd = gp$alpha_sd,
     gp_type = data.table::fcase(
-    gp$kernel == "se", 0,
-    gp$kernel == "matern", 1,
-    default = 0
+      is.infinite(gp$matern_order), 0,
+      gp$matern_order == 1 / 2, 1,
+      gp$matern_order == 3 / 2, 2,
+      default = 3
```

Comment:
This follows the previous pattern but do we want to make it instead take 5/2 explicitly and if given something it doesn't recognise have a meaningful error?